data/schemas/files.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Files",
+    "description": "Language specific list of source textual files",
+    "type": "array",
+    "items": {
+        "additionalProperties": false,
+        "required": [
+            "category",
+            "textname",
+            "filenr"
```

Comment:
not sure if I read the scheme correctly but 'displayName' should be a required field as well. AFAIK the frontend throws errors when NULL value is returned on the menu etc.

Reply:
Yes. I missed it